tests/ui/arch/atomics/atomic_xor.rs

```diff
@@ -0,0 +1,19 @@
+// build-pass
+
+#![allow(incomplete_features)]
+#![feature(generic_const_exprs)]
```

Comment:
Do all of these tests need `#![feature(generic_const_exprs)]`?

Reply:
Seems like these are redundant, they have been removed